### ISSUE TITLE
AppImage for Linux

### ISSUE THREAD

**Issue by probonopd** [jackd248/temps#15]
Please provide an [AppImage](http://appimage.org/) for Linux. This will allow temps to run on many Linux distributions at once. `electron-builder` has support for it, so should be really easy:

1. Edit package.json to use the latest electron-builder.

2. Edit package.json — add after "build": { ... }:

```
    , "linux": {
      "target": ["AppImage"]
    },

```
Then, `npm install` or `npm run dist or npm run dist -- --linux=AppImage` if you want to explicitly set target and do not edit package.json.

**Comment by Ygarr** [comment]
snap image is also suitable and commonly convenient

**Comment by probonopd** [comment]
@Ygarr yes, but so far almost only Ubuntu comes with it out-of-the-box, whereas AppImages run almost everywhere.
